shared/bundle_analysis/models.py

```diff
@@ -112,7 +112,6 @@ def __enter__(self):
 
     def __exit__(self, type, value, traceback):
         self.session.close()
-        return True
```

Comment:
Returning True here actually suppresses exceptions being thrown by the DB session context manager